modules/GraphvizLayout/nb-configuration.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this file is superfluous and only netbeans related. Can you remove it from the Git Index?